graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -926,8 +934,18 @@ def key_is_mask(key: str):
                     type_name,
                 )
                 repartitioner.modify_metadata_for_flat_arrays(node_label_files)
-                for ntype_data_entry_key, ntype_data_entry_val in ntype_data_dict.items():
-                    if key_is_mask(ntype_data_entry_key):
+                for (
+                    ntype_data_entry_key,
+                    ntype_data_entry_val,
+                ) in ntype_data_dict.items():
+                    if "ntype_to_label_masks" in graph_info:
+                        ntype_label_masks: list[str] = graph_info["ntype_to_label_masks"][type_name]
```

Comment:
How do you handle the case when there are more than one training mask on a node type?

Reply:
During graph construction all the mask names are gathered in a list for each node type. The structure is:


```python
# graph_info
{
    "ntype_to_label_masks": {
        "paper": [
            "class_train_mask",
            "class_val_mask",
            "class_test_mask",
            "reg_train_mask",
            "reg_val_mask",
            "reg_test_mask"
        ]
    }
}
```

and similarly for each edge type.

Reply:
I'll add this info in graph_info documentation.